packages/digitalocean-worker/infrastructure/providerconfig.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: digitalocean.crossplane.io/v1beta1
+kind: ProviderConfig
```

Comment:
To DRY out our ProviderConfigs, we can just move them to the modules, and with the `namePrefix` in Kusomization file, we grant that each cluster would have its own separate ProviderConfig resource.

Same as here: https://github.com/kubezero/kubezero/blob/main/modules/aws/eks/cluster/provider-config.yaml